integration/spring-pulsar/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot'
+    id 'org.springframework.boot.aot'
+    id 'org.springframework.aot.smoke-test'
+    id 'org.graalvm.buildtools.native'
+}
+
+repositories {
+    mavenLocal()
```

Comment:
Instead of setting `mavenLocal()` take a look at the [`fromMavenLocal`](https://github.com/spring-projects/spring-aot-smoke-tests/blob/90e784470f6f0a4b2e328df3ea1fda415fea6bf5/gradle/plugins/aot-smoke-test-plugin/src/main/java/org/springframework/aot/gradle/AotSmokeTestPlugin.java#L81) property.